Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -478,7 +478,7 @@ recorded otherwise:
   }
 }
 
-#### Converting XCTSkip(), XCTSkipIf(), and XCTSkipUnless() calls
+#### Convert XCTSkip(), XCTSkipIf(), and XCTSkipUnless() calls
```

Comment:
Delete; can't have subheadings within a section. Also, don't include symbols in headings.

Reply:
As above?